content/microcontrollers/esp8266-nodemcu.md

```diff
@@ -81,7 +81,7 @@ Now you should be able to flash your board as follows:
 - Build and flash your TinyGo code using the `tinygo flash` command. This command flashes the ESP826 with the blinky1 example:
 
     ```
-    tinygo flash -target=esp8266 examples/blinky1
+    tinygo flash -target=nodemcu examples/blinky1
```

Comment:
```suggestion
    tinygo flash -target=nodemcu -port=/dev/ttyUSB examples/blinky1
```

Reply:
For windows, the description would be something like `-port=COM5`.

Reply:
I think the esptool can locate the port itself on Windows in similar fashion to macOS. If not, let's handle in subsequent PR so we can get everything else done.